cryptography/src/lib.rs

```diff
@@ -127,13 +128,36 @@ pub trait BatchScheme {
     fn verify<R: RngCore + CryptoRng>(self, rng: &mut R) -> bool;
 }
 
-/// Error Digest manipulations must rely on.
+/// Errors that can occur when interacting with cryptographic primitives.
 #[derive(Error, Debug)]
 pub enum Error {
     #[error("invalid digest length")]
     InvalidDigestLength,
 }
 
+/// Byte array representing an arbitrary hash digest.
+pub trait Digest:
```

Comment:
This allows crates that only need to handle `Digest` to do so (otherwise it implies that they might also be hashing content).